### ISSUE TITLE
Website/A11y - Skip links are not mentioned at https://clarity.design/documentation/app-layout

### ISSUE THREAD

**Issue by clane** [vmware/clarity#3764]
## How to reproduce

1.  Go to https://clarity.design/documentation/app-layout
2.  Observe that there is no mention of the use of "skip links" to allow keyboard-only users the ability to move focus past groups of links or other controls such as those found in navigation.

## Expected behavior
The layout documentation example should include skip links, and the purpose of skip links should be explained. 

## Versions

**App**

* Angular: NA
* Node: NA
* Clarity: NA

**Device:**

* Type: Any
* OS: Any
* Browser Any 
* Version 

## Additional notes

For more information please review these resources.

Success Criterion 2.4.1 Bypass Blocks
https://www.w3.org/TR/WCAG21/#bypass-blocks

Understanding Success Criterion 2.4.1: Bypass Blocks
https://www.w3.org/WAI/WCAG21/Understanding/bypass-blocks.html

**Comment by clane** [comment]
I am unable to label this issue, if I could I would have applied the Website and Accessibility labels.

**Comment by mathisscott** [comment]
This is content that should go into an accessibility section. It's not something the Clarity devs should produce and maintain.

**Comment by gnomeontherun** [comment]
Clarity has a vision for including some accessibility content, but this is a specific detail that we don't need to track. Closing as something that is outside of Clarity's documentation or may be included as part of a generic accessibility guide.
